child_compassion/mappings/child_lifecycle_mapping.py

```diff
@@ -0,0 +1,59 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2016 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Philippe Heer <heerphilippe@msn.com>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+from openerp.addons.message_center_compassion.mappings.base_mapping import \
+    OnrampMapping
+
+
+class ICPMapping(OnrampMapping):
+    ODOO_MODEL = 'compassion.child.ble'
+    MAPPING_NAME = "new_child_lifecyle"
```

Comment:
remove mapping_name or specify it in the action xml, otherwise it won't find the mapping.